packages/ui/src/views/layouts/SandboxRoot.tsx

```diff
@@ -22,13 +21,14 @@ export interface SandboxRootProps {
   currentUser: TypesUser | undefined
   logout?: () => void
   useNav: () => NavState
-  t: TFunction
+  useThemeStore: () => IThemeStore
+  useTranslationStore: () => TranslationStore
 }
 
-export const SandboxRoot = ({ currentUser, useNav, t, logout }: SandboxRootProps) => {
+export const SandboxRoot = ({ currentUser, useNav, logout, useThemeStore, useTranslationStore }: SandboxRootProps) => {
```

Comment:
Looks like this causes some type issues in the project

Reply:
yup, checking this